### PR TITLE
fix: rework example project

### PR DESCRIPTION
**Pull request by ni507**

### CODE REVIEW

File: proto-files/hello.proto [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// based on https://grpc.io/docs/guides/concepts.html
+
+syntax = "proto2";
+
+package hello;
+
+service HelloService {
```
Comment (CommonGuy) [inline proto-files/hello.proto:7]: Do we really need this?

File: example-project/grpc/BidiHello.krop [hunk 2]
```diff
@@ -8,5 +8,6 @@
     }
   ],
   "operationType": "duplexStreaming",
-  "invokerName": "grpc"
+  "invokerName": "grpc",
```
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/grpc/BidiHello.krop:11]: Added more requests.

File: example-project/grpc/LotsOfGreetings.krop [hunk 3]
```diff
@@ -8,5 +8,6 @@
     }
   ],
   "operationType": "clientStreaming",
-  "invokerName": "grpc"
+  "invokerName": "grpc",
```
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/grpc/LotsOfGreetings.krop]: Added more requests.

File: example-project/grpc/BidiHello.krop [hunk 4]
```diff
@@ -8,5 +8,6 @@
     }
   ],
   "operationType": "duplexStreaming",
```
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/grpc/BidiHello.krop]: Renamed operations and folder.

File: example-project/rest/path params {code}/status-2.krop
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/rest/path params {code}/status-2.krop:1]: Renamed folders and remove second operation.

File: example-project/rest/path params {code}/status.krop
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/rest/path params {code}/status.krop:1]: Changed request message.

File: example-project/Kreya features/auth/SayHello with Auth-request.json
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/Kreya features/auth/SayHello with Auth-request.json:1]: Changed comment.

File: example-project/Kreya features/scripting/Scripting gRPC.js
[the remark this answers is not in the document]
Comment (ni507) [reply example-project/Kreya features/scripting/Scripting gRPC.js:1]: Renamed folders.

**Pull request merged**